cypress/e2e/dashboard.spec.ts

```diff
@@ -0,0 +1,270 @@
+import {
+  addCollaborator,
+  getOpenStagingButton,
+  removeOtherCollaborators,
+  visitE2eEmailTestRepo,
+} from "../utils"
+
+const getReviewRequestButton = () => cy.contains("button", "Request a Review")
+const getSubmitReviewButton = () => cy.contains("button", "Submit Review")
+
+const selectReviewer = (email: string) => {
+  cy.get("div[id^='react-select']").contains(E2E_EMAIL_COLLAB.email).click()
+}
+const removeReviewers = () => cy.get('div[aria-label^="Remove"]').click()
+
+const TITLE_INPUT_SELECTOR =
+  "input[placeholder='Title your request'][name='title']"
+const DESCRIPTION_TEXTAREA_SELECTOR =
+  "textarea[name='description'][placeholder='Briefly describe the changes you’ve made, and add a review deadline (if applicable)']"
+
+const getAddReviewerDropdown = () =>
+  cy.contains("Select Admins to add as reviewers").parent()
+
+const goToWorkspace = () => cy.contains("a", "Edit site").click()
+
+const REVIEW_MODAL_SUBTITLE =
+  "An Admin needs to review and approve your changes before they can be published"
+
+describe("dashboard flow", () => {
+  beforeEach(() => {
+    cy.setupDefaultInterceptors()
+    cy.setEmailSessionDefaults("Email admin")
+    visitE2eEmailTestRepo()
+  })
+  describe("no review requests", () => {
+    beforeEach(() => {
+      closeReviewRequests()
+    })
+    it('should open the staging site on click of the "Open staging" button', () => {
+      // Act
+      getOpenStagingButton()
+        .should("have.attr", "href", E2E_EMAIL_REPO_STAGING_LINK)
+        .should("have.attr", "target", "_blank")
+    })
+    it('should open the "Request a Review" modal on click of the "Request a Review" button', () => {
+      // Act
+      getReviewRequestButton().click()
+
+      // Assert
+      cy.contains(REVIEW_MODAL_SUBTITLE).should("be.visible")
+    })
+    it.skip("should be able to navigate to the staging site using the dropdown button", () => {
+      throw new Error("Not implemented")
+    })
+    it.skip("should be able to navigate to the production site using the dropdown button", () => {
+      throw new Error("Not implemented")
+    })
+    it.skip('should navigate to the isomer guide on click of the "Get help" button', () => {
+      throw new Error("Not implemented")
+    })
+    it.skip("should navigate to the settings page when manage site settings is clicked", () => {
+      throw new Error("Not implemented")
+    })
+    it.skip("should navigate to the workspace when edit site is clicked", () => {
+      // Arrange
+      // NOTE: There shouldn't be an alert
+      // Act
+      // Assert
+      throw new Error("Not implemented")
+    })
+  })
+  describe("create review request", () => {
+    beforeEach(() => {
+      // NOTE: We want to start this test suite on a clean slate
+      // and prevent other tests from interfering
+      removeOtherCollaborators()
+    })
+
+    it("should not be able to create a review request when the site has 1 collaborator", () => {
```

Comment:
should we also add a 
"should not be able to add contributor as a reviewer"? This seems like a critical access flow control we should test for

Reply:
see above comment - maybe i can change the wording of the test to make it clear that we're also trying to add a contributor